### DISCUSSION TITLE
WebView JS Console  triggle

### DISCUSSION THREAD

**Discussion by stonewjr** [tauri-apps/wry#335] [General]
How to disable WebView JS Console be triggle in production release

**Comment by amrbashir** [comment]
Devtools are disabled in release build and if you want to enable them, use a debug version `tauri build --debug`

**Comment by nothingismagick** [comment]
What's the trouble with triggles?

**Comment by stonewjr** [comment]
thanks response

I test out and got it

nothingismagick ***@***.***> 於 2021年7月15日 週四 上午12:25寫道：
